be/src/exprs/jit/ir_helper.h

```diff
@@ -30,7 +30,7 @@ struct LLVMDatum {
     llvm::Value* value = nullptr;     ///< Represents the actual value of the datum.
     llvm::Value* null_flag = nullptr; ///< Represents the nullity status of the datum.
 
-    LLVMDatum(llvm::IRBuilder<>& b) { null_flag = llvm::ConstantInt::get(b.getInt8Ty(), 0); }
+    LLVMDatum(llvm::IRBuilder<>& b) { null_flag = llvm::ConstantInt::get(b.getInt8Ty(), 0, false); }
```

Comment:
why add `false`?

Reply:
In LLVM, all types are unsigned, making an explicit declaration here. This is primarily to differentiate from signed numbers, as when creating an LLVM ConstantInt using literals, a signed 0 will be misconstrued as an unsigned 0, causing an error.